tox.ini

```diff
@@ -0,0 +1,144 @@
+# Differences from the starbase tox.ini:
```

Comment:
Do I need to install `tox-ignore-env-name-mismatch` manually?  Should it be in `requirements-devel.txt`?

All my tox runs log:
 `ROOT: will run in automatically provisioned tox, host /home/developer/dev/snapcraft/venv/bin/python is missing [requires (has)]: tox-ignore-env-name-mismatch==0.2.0.post2`

Reply:
It should reuse the same `[project_dir]/.tox/.tox` environment. If we want to put in dev requiremnts I'm happy to do so, but tbh you don't even need to be in a venv to run the tox stuff. With this change you can even just use tox from apt regardless of the version and it'll provision all the environments for you. 

(Which also means we could even remove tox from the dev requirements...)

Reply:
Interesting.  So I safely can ignore that warning?

Reply:
Yeah, it's more of an info thing than a warning, but it's intended from the way I did the config. There doesn't appear to be a way to tell tox not to display it.